cmds/bzImage/bzImage_test.go

```diff
@@ -0,0 +1,182 @@
+			skip:   uskip,
+		},
+		{
+			args:   []string{"initramfs", "bzImage", "init.cpio", "placeholder"},
+			status: 0,
+			out:    "",
+		},
+		{
+			args:   []string{},
+			status: 1,
+			out:    cmdUsage + "\n",
+			skip:   uskip,
+		},
+		{
+			args:   []string{"dump", "bzImage"},
+			status: 0,
+			out:    "MBRCode:0xea0500c0078cc88ed88ec08ed031e4fbfcbe2d00ac20c07409b40ebb0700cd10ebf231c0cd16cd19eaf0ff00f0557365206120626f6f74206c6f616465722e0d0a0a52656d6f7665206469736b20616e6420707265737320616e79206b657920746f207265626f6f742e2e2e0d0a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\nExtRamdiskImage:0x00\nExtRamdiskSize:0x00\nExtCmdlinePtr:0x00\n_:0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\nSetupSects:0x1e\nRootFlags:0x01\nSyssize:0xb51d\nRamSize:0x00\nVidmode:0xffff\nRootDev:0x00\nBootsectormagic:0xaa55\nJump:0x66eb\nHeaderMagic:0x48647253\nProtocolversion:0x20d\nRealModeSwitch:0x00\nStartSys:0x1000\nKveraddr:0x3140\nTypeOfLoader:0x00\nLoadflags:0x01\nSetupmovesize:0x8000\nCode32Start:0x100000\nRamDiskImage:0x00\nRamDiskSize:0x00\nBootSectKludge:0x00000000\nHeapendptr:0x5320\nExtLoaderVer:0x00\nExtLoaderType:0x00\nCmdlineptr:0x00\nInitrdAddrMax:0x7fffffff\nKernelalignment:0x200000\nRelocatableKernel:0x00\nMinAlignment:0x15\nXLoadFlags:0x01\nCmdLineSize:0x7ff\nHardwareSubArch:0x00\nHardwareSubArchData:0x00\nPayloadOffset:0x255\nPayloadSize:0x9532c\nSetupData:0x00\nPrefAddress:0x1000000\nInitSize:0x6e0000\nHandoverOffset:0x00\n",
+		},
+		{
+			args:   []string{"initramfs"},
+			status: 1,
+			out:    cmdUsage + "\n",
+			skip:   uskip,
+		},
+		{
+			args:   []string{"initramfs", "a", "b", "c", "too many"},
+			status: 1,
+			out:    cmdUsage + "\n",
+			skip:   uskip,
+		},
+		{
+			args:   []string{"initramfs", "a", "b", "c"},
+			status: 1,
+			out:    "open a: no such file or directory\n",
+			skip:   uskip,
+		},
+		{
+			args:   []string{"initramfs", "bzImage", "b", "c"},
+			status: 1,
+			out:    "open b: no such file or directory\n",
+			skip:   uskip,
+		},
+	}
+)
+
+func TestSimple(t *testing.T) {
+	tmpDir, err := ioutil.TempDir("", "bzImage")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(tmpDir)
+	// adjust the args for the output test.
+	// Create a bogus path in our tmpDir
+	tests[0].args[3] = filepath.Join("zz", "zz", "zz")
```

Comment:
If you move the table `b` to be a local variable defined here, you won't need to swap out the placeholder values.

Reply:
done